extra/exports

```diff
@@ -390,6 +390,7 @@ say_parse_logger_type
 say_set_log_format
```

Comment:
> NO_CHANGELOG=BUGFIX

It is bugfix, so what? It is the user visible change, which makes the behaviour different from one that was released in 2.10.0.

> NO_TEST=BUGFIX

But you have added a test. It seems, you don't need `NO_TEST`.

Reply:
Fixed.

Reply:
Thanks! (Please, update or, better, strip it from the pull request description.)